lib/user.cpp

```diff
@@ -219,6 +240,11 @@ bool User::isGuest() const
     return *it == ':';
 }
 
+int User::hue() const
+{
+    return d->hue();
```

Comment:
Just inline it, don't bother making a trivial method.

Reply:
changed both for `User::hue` and `User::hueF`

Reply:
modifed my changes thanks to your help